packages/std/src/math/uint256.rs

```diff
@@ -99,6 +99,23 @@ impl Uint256 {
         Uint256(U256(words))
     }
 
+    pub const fn from_uint128(num: Uint128) -> Self {
+        let num = num.u128();
+        let bytes = num.to_le_bytes();
+        let words: [u64; 4] = [
+            u64::from_le_bytes([
+                bytes[0], bytes[1], bytes[2], bytes[3], bytes[4], bytes[5], bytes[6], bytes[7],
+            ]),
+            u64::from_le_bytes([
+                bytes[8], bytes[9], bytes[10], bytes[11], bytes[12], bytes[13], bytes[14],
+                bytes[15],
+            ]),
+            0,
+            0,
+        ];
+        Self(U256(words))
```

Comment:
Wouldn't it be nice to reuse `Self::from_be_bytes` here? This reduces the places where we mess with U256 internals.

Reply:
Yeah, good idea